src/content/reference/okteto-cli.mdx

```diff
@@ -21,17 +23,9 @@ Use `okteto command --help` for information on a specific command. The synopsis
 
 The amount of information outputted (defaults to _warn_).
 
-## Advanced configuration
```

Comment:
This is very advanced, and only related to Build, to be at the top of the Okteto CLI docs. Moved at the end of the Build Service docs